EXOSIMS/Prototypes/ZodiacalLight.py

```diff
@@ -341,80 +361,68 @@ def calcfZmin(self, sInds, Obs, TL, TK, mode, hashname, koMap=None, koTimes=None
                 Selected observing mode
             hashname (str):
                 hashname describing the files specific to the current json script
-            koMap (~numpy.ndarray(bool)):
+            koMap (~numpy.ndarray(bool), optional):
                 True is a target unobstructed and observable, and False is a
                 target unobservable due to obstructions in the keepout zone.
-            koTimes (~astropy.time.Time(~numpy.ndarray(float))):
-                Absolute MJD mission times from start to end in steps of 1 d
+            koTimes (~astropy.time.Time(~numpy.ndarray(float)), optional):
+                Absolute MJD mission times from start to end, in steps of 1 d as default
 
         Returns:
-            list:
-                list of local zodiacal light minimum and times they occur at
-                (should all have same value for prototype)
+            tuple:
+                fZmins[n, TL.nStars] (~astropy.units.Quantity(~numpy.ndarray(float))):
+                    fZMap, but only fZmin candidates remain. All other values are set to
+                    the maximum floating number. Units are 1/arcsec2
+                fZtypes [n, TL.nStars] (~numpy.ndarray(float)):
+                    ndarray of flags for fZmin types that map to fZmins
+                    0 - entering KO
+                    1 - exiting KO
+                    2 - local minimum
+                    max float - not a fZmin candidate
         """
 
         # Generate cache Name
         cachefname = hashname + "fZmin"
 
         # Check if file exists
         if os.path.isfile(cachefname):  # check if file exists
-            self.vprint("Loading cached fZQuads from %s" % cachefname)
-            # load from cache. fZQuads has the form tmpDat len sInds, tmpDat[0] len
-            # number of ko enter/exits and localmin occurences, tmpDat[0,0] form
-            # [type,fZvalue,absTime]
-            with open(cachefname, "rb") as f:  #
-                fZQuads = pickle.load(f)
-                # Convert Abs time to MJD object
-                for i in np.arange(len(fZQuads)):
-                    for j in np.arange(len(fZQuads[i])):
-                        fZQuads[i][j][3] = Time(
-                            fZQuads[i][j][3], format="mjd", scale="tai"
-                        )
-                        fZQuads[i][j][1] = fZQuads[i][j][1] / u.arcsec**2.0
-
-            return [fZQuads[i] for i in sInds]
+            self.vprint("Loading cached fZmins from %s" % cachefname)
+            with open(cachefname, "rb") as f:  # load from cache
+                tmp1 = pickle.load(f, encoding="latin1")

```

Comment:
the encoding is probably unnecessary here as we're not trying for python2 compatibility.